sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/Samples/Sample2_AnalyzeConversationWithOptions.cs

```diff
@@ -20,15 +20,27 @@ public void AnalyzeConversationWithOptions()
 
 #if SNIPPET
             ConversationsProject conversationsProject = new ConversationsProject("Menu", "production");
-            AnalyzeConversationOptions options = new AnalyzeConversationOptions(
-                "We'll have 2 plates of seared salmon nigiri.");
-
-            Response<AnalyzeConversationResult> response = client.AnalyzeConversation(conversationsProject, options);
+            AnalyzeConversationOptions options = new AnalyzeConversationOptions(){
+                IsLoggingEnabled = true,
+                Verbose = true,
+                Language = "en"
```

Comment:
1. Since the server default is "en" anyway, is it worth specifying it?
2. Might be good to add a sample showing how to set the `ConverationsClientOptions.DefaultLanguage` as well - or did we do that only for Question Answering?

Reply:
1. I'll remove it, I thought it might be useful to show some of the attributes inside our Options bag.
2. We haven't done that for Conversations yet. I believe it's one of the pending issues.